.bazelrc

```diff
@@ -0,0 +1,176 @@
+
+build:cuda --repo_env TF_NEED_CUDA=1
+# "sm" means we emit only cubin, which is forward compatible within a GPU generation.
+# "compute" means we emit both cubin and PTX, which is larger but also forward compatible to future GPU generations.
+build:cuda --crosstool_top=@local_config_cuda//crosstool:toolchain
+build:cuda --@local_config_cuda//:enable_cuda
+build:cuda --@org_tensorflow//tensorflow/compiler/xla/python:enable_gpu=true
+build:cuda --define=xla_python_enable_gpu=true
+build:cuda --cxxopt=-DXLA_CUDA=1
+
+build:acl --define==build_with_acl=true
+
+build:nonccl --define=no_nccl_support=true
+
+build:linux --config=posix
+build:linux --copt=-Wno-unknown-warning-option
+
+# Suppress all warning messages.
+build:short_logs --output_filter=DONT_MATCH_ANYTHING
+
+#build:tpu --@org_tensorflow//tensorflow/compiler/xla/python:enable_tpu=true
+build:tpu --define=with_tpu_support=true
+
+#########################################################################
+# RBE config options below.
+# Flag to enable remote config
+common --experimental_repo_remote_exec
+#########################################################################
+
+# Load rc file with user-specific options.
+try-import %workspace%/.bazelrc.user
+
+# Compile database generation config.
+build:compdb --features=-layering_check
+
+# Test requires Java.
+test --java_runtime_version=remotejdk_11
+
+# Coverage requires Java and GCC.
+coverage --config=coverage
+coverage --build_tests_only
+build:coverage --java_runtime_version=remotejdk_11
+build:coverage --copt=-DNDEBUG
+build:coverage --combined_report=lcov
+build:coverage --strategy=TestRunner=sandboxed,local
+build:coverage --strategy=CoverageReport=sandboxed,local
+build:coverage --experimental_use_llvm_covmap
+build:coverage --collect_code_coverage
+build:coverage --test_tag_filters=-nocoverage
+build:coverage --action_env=CC=gcc
+build:coverage --action_env=CXX=g++
+
+############################################################################
+############## TensorFlow .bazelrc greatest hits ###########################
+############################################################################
+
+# Modular TF build options
+build:dynamic_kernels --define=dynamic_loaded_kernels=true
+build:dynamic_kernels --copt=-DAUTOLOAD_DYNAMIC_KERNELS
+build --define=tf_api_version=2 --action_env=TF2_BEHAVIOR=1
+
+# Default paths for TF_SYSTEM_LIBS
+build:linux --define=PREFIX=/usr
+build:linux --define=LIBDIR=$(PREFIX)/lib
+build:linux --define=INCLUDEDIR=$(PREFIX)/include
+build:linux --define=PROTOBUF_INCLUDE_PATH=$(PREFIX)/include
+
+# On linux, we dynamically link small amount of kernels
+build:linux --config=dynamic_kernels
+
+# For projects which use TensorFlow as part of a Bazel build process, putting
+# nothing in a bazelrc will default to a monolithic build. The following line
+# opts in to modular op registration support by default.
+build --define framework_shared_object=true
```

Comment:
General comment: in case we introduce / modify environment variables, we should keep [configuration.yaml](https://github.com/pytorch/xla/blob/master/configuration.yaml) up to date.

Reply:
Agreed.

I would not introduce any env variables, I would remove all of them eventually. Configuration should happen through ansible/bazel in the future. We should only keep the ones that are required to configure downstream stuff. wdyt?